cmd/root.go

```diff
@@ -40,7 +53,8 @@ func Execute() {
 }
 
 func init() {
-
 	cobra.OnInitialize()
-	rootCmd.Flags().StringP("token", "t", "", "Enter a valid telegram bot token")
+	rootCmd.Flags().StringP("bot token", "b", "", "Enter a valid telegram bot token")
```

Comment:
flags are normally separated with hyphens so the command would look like `statusbot --bot-token`